File: cfgs/java_cfgs_foo/CHECK_NUMBER_IS_PERFECT_SQUARE_USING_ADDITIONSUBTRACTION.java
Nodes
1;special;;;
2;statement;int sum = 0;14;
3;statement;int i = 1;14;
4;conditional;foo_1 < foo_2;15;
5;statement;sum += foo_1;17;
6;conditional;foo_1 == foo_2;18;
7;statement;return true;18;
8;exit;;;
9;statement;i += 2;16;
10;statement;return false;20;
Edges
0;2;
1;3;
2;4;
3;5;10;
4;6;
5;7;9;
6;8;
7;
8;4;
9;8;

File: cfgs/pythoncfgs/CHECK_NUMBER_IS_PERFECT_SQUARE_USING_ADDITIONSUBTRACTION.py
Nodes
1;statement;foo_1 = 1;8;
2;statement;foo_1 = 0;9;
3;conditional;foo_1 < foo_2;10;
4;statement;return false;15;
5;statement;foo_1 += foo_2;11;
6;conditional;foo_1 == foo_2;12;
7;statement;foo_1 += 2;14;
8;statement;return true;13;
9;exit;;
Edges
0;2;
1;3;
2;4;5;
3;9;
4;6;
5;7;8;
6;3;
7;9;
8;
